### ISSUE TITLE
About the operation steps

### ISSUE THREAD

**Issue by seven-linglx** [jinfagang/bboxer#1]
In my opinion, your explain is not clear enough. it is better insert a sentence.

1 run 
2 draw box at the image
3 input the class like 1 (click number 1 in keyboard)
4 click s to save
5 processing next image

**Comment by lucasjinreal** [comment]
Yeah, you probably got it. I will change it in next version update.
